data-reconciliation-app/Makefile

```diff
@@ -108,4 +107,4 @@ generate-access-token: ## 🔐 Generate and access token
 clean: ## 🧹 Clean the working folders created during build/demo
 	@rm -rf .venv_ccf_sandbox
 	@rm -rf workspace
-	@rm -rf dist
+	@rm -rf dist
```

Comment:
Can we ensure that we also have a blank line at the end of the file

Reply:
Done